proto/documents/v3alpha/documents.proto

```diff
@@ -218,20 +371,57 @@ message DocumentListItem {
   // User-defined metadata attributes of the document.
   map<string, string> metadata = 3;
 
-  // Every author ID who has modified this document's version.
+  // Every author ID who has ever made changes to this document.
   repeated string authors = 4;
 
   // Time when the document was created.
   google.protobuf.Timestamp create_time = 5;
 
   // Time when the document was updated.
+  // I.e. timestamp of the most recent change.
   google.protobuf.Timestamp update_time = 6;
 
   // ID of the genesis change of this document.
   string genesis = 8;
 
   // Current version of the document.
   string version = 9;
+
+  // Breadcrumbs of this document.
+  // The number of items in this list corresponds to the number of segments
+  // in the path of the document minus 1 (the document itself).
+  //
+  // E.g. for a document "/cars/jp/honda" there will be 2 breadcrumbs: "cars" and "jp".
```

Comment:
we also need the site name, not just the names of the parent docs

Reply:
I thought we'd be already having the site name in memory when calling methods for listing documents?

Reply:
I'll add the change ID